blockonomics-woocommerce.php

```diff
@@ -40,6 +40,9 @@
 require_once( ABSPATH . 'wp-admin/includes/upgrade.php' );
 require_once ABSPATH . 'wp-admin/install-helper.php';
 
+// Define constant for Plugin Directory
+define( 'BLOCKONOMICS_PLUGIN_DIR', plugin_dir_path( __FILE__ ) );
```

Comment:
This is no longer in use and can be removed

Reply:
Done